packages/trie/test/stream.spec.ts

```diff
@@ -156,21 +156,21 @@ tape('db stream test', function (tester) {
     },
   ] as BatchDBOp[]
 
-  const expectedNodes = {
+  /*const expectedNodes = {
     '3c38d9aa6ad288c8e27da701e17fe99a5b67c8b12fd0469651c80494d36bc4c1': true,
     d5f61e1ff2b918d1c2a2c4b1732a3c68bd7e3fd64f35019f2f084896d4546298: true,
     e64329dadee2fb8a113b4c88cfe973aeaa9b523d4dc8510b84ca23f9d5bfbd90: true,
     c916d458bfb5f27603c5bd93b00f022266712514a59cde749f19220daffc743f: true,
     '2386bfb0de9cf93902a110f5ab07b917ffc0b9ea599cb7f4f8bb6fd1123c866c': true,
-  } as any
+  } as any*/
 
   it('should populate trie', async function (t) {
     trie.checkpoint()
     await trie.batch(ops)
     t.end()
   })
 
-  it('should only fetch nodes in the current trie', function (t) {
+  /*it('should only fetch nodes in the current trie', function (t) {
```

Comment:
still commented out

Reply:
This test doesn't work any more since `_scratchReadStream` related, will remove.